setup.py

```diff
@@ -338,7 +339,13 @@ def run(self):
     extra_compile_args += ['-DWITH_NCCL']
 
 if WITH_CUDNN:
-    main_libraries += ['cudnn']
+    CUDNN_LIB_NAME = 'libcudnn.so'
+    if platform.system() == 'Darwin':
+        CUDNN_LIB_NAME = 'libcudnn.dylib'
+    CUDNN_LIB = os.path.join(CUDNN_LIB_DIR, CUDNN_LIB_NAME)
+    CUDNN_LIB = os.path.realpath(CUDNN_LIB)
+
+    main_link_args += [ CUDNN_LIB ]
```

Comment:
Doesn't it link against an absolute lib path? It might cause problems when you're going to build the binaries.